haas/model.py

```diff
@@ -40,17 +42,31 @@
 
 db = SQLAlchemy(app)
 
+BigIntegerType = BigInteger()
+BigIntegerType = BigIntegerType.with_variant(
+        sqlite.INTEGER(), 'sqlite')
+
```

Comment:
Why can't we just do?

`BigIntegerType = BigInteger().with_variant(sqlite.INTEGER(), 'sqlite')`

Same thing in other places too